ikasaneip/rest/rest-module/src/main/java/org/ikasan/rest/module/InDoubtTransactionsApplication.java

```diff
@@ -0,0 +1,116 @@
+package org.ikasan.rest.module;
+
+import org.ikasan.rest.module.dto.ErrorDto;
+import org.ikasan.rest.module.dto.InDoubtTransactionDto;
+import org.ikasan.rest.module.dto.TableRowCountDto;
+import org.ikasan.rest.module.util.UserUtil;
+import org.ikasan.spec.persistence.model.InDoubtTransaction;
+import org.ikasan.spec.persistence.service.GeneralDatabaseService;
+import org.ikasan.spec.persistence.service.InDoubtTransactionService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+/**
+ * The InDoubtTransactionsApplication class is a RESTful API controller for handling
+ * in-doubt transactions.
+ */
+@RequestMapping("/rest/transaction/inDoubt")
+@RestController
+public class InDoubtTransactionsApplication
```

Comment:
Should we also add Controller to allow call service to  CommitAll and RollBackAll?

Reply:
@andrzej-majewski have added feature to commit and rollback all